### ISSUE TITLE
Close Modal Button Missing Label

### ISSUE THREAD

**Issue by Scott-James-Hurley** [UserOfficeProject/user-office-project-issue-tracker#1683]
The close button for some modals e.g. the edit person, add instrument/call to tag and assign experiment scientist lack a label. This means its purpose may not be obvious to screen reader users. We should add a label that will be displayed when the button is focussed and read out by screen readers
